app/src/main/res/values/strings.xml

```diff
@@ -290,6 +290,13 @@
     <string name="maps_markers_badges">Badges</string>
     <string name="maps_markers_badges_with_icons">With icons</string>
     <string name="map_markers">Map markers</string>
+    <string name="map_marker_price_1">£195</string>
```

Comment:
We can create an array resource, or not to create a resource at all and just use build-in currency formatters

Reply:
I'm not sure how this is different to any other component we have where we just use standard string resources for different examples, so i'd rather not overcomplicate things